server/src/main/java/io/druid/server/coordinator/DruidCoordinatorRuntimeParams.java

```diff
@@ -206,7 +205,7 @@ public Builder buildFromExisting()
       this.stats = new CoordinatorStats();
       this.coordinatorDynamicConfig = new CoordinatorDynamicConfig.Builder().build();
       this.balancerReferenceTimestamp = DateTime.now();
-      this.strategyFactory = new CostBalancerStrategyFactory(1);
+      this.strategyFactory = new CostBalancerStrategyFactory(1, 0.001F);
```

Comment:
the default should be made a constant somewhere instead of being defined in multiple places